src/Features/Admin/Users/actions.js

```diff
@@ -0,0 +1,7 @@
+import { CREATE_USER, DELETE_USER_BY_ID, FETCH_USERS, FETCH_USER_BY_ID, UPDATE_USER_BY_ID } from './ActionTypes';
+
+export const createUser = (data) => ({ type: CREATE_USER, payload: data });
+export const fetchUsers = (data) => ({ type: FETCH_USERS, payload: data });
+export const fetchUserById = (data) => ({ type: FETCH_USER_BY_ID, payload: data });
```

Comment:
Why this action is still here when you are using react query .. please remove unused code ..